common4j/src/main/com/microsoft/identity/common/java/commands/parameters/AcquirePrtSsoTokenCommandParameters.java

```diff
@@ -0,0 +1,53 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.java.commands.parameters;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * This is a set of command parameters for generating SSO tokens.  This is only being
+ * created to conform to the interface for the packages that talk to the broker, since
+ * it is mostly irrelevant for any api usage (there is no non-broker implementation of
+ * this).
+ */
+@Getter
+@SuperBuilder(toBuilder = true)
+@EqualsAndHashCode(callSuper = true)
+@Accessors(prefix = "m")
+public class AcquirePrtSsoTokenCommandParameters extends CommandParameters {
+    /**
+     * The sso url parameter for the token being fetched.
+     */
+    private final String mSsoUrl;
+    /**
+     * The account name for the token being fetched.
+     */
+    private final String mAccountName;
```

Comment:
What is this meant to represent? UPN or something else?

Reply:
I thought account names were inclusive of UPN, but that they included more.

Reply:
It can be email too.

Reply:
I was more interested in whether this is meant to be the `accountName` as in the `IBrokerAccount` interface?

Reply:
you may want to just take the whole `IAccountRecord` object as input here to the command parameters

Reply:
since this operation is a silent operation, the user probably has the account object...or may be it doesn't matter for us

Reply:
I think this field is probably required (and should be marked with NonNull as such)

Reply:
This is now an IAccount implementation.